docs/consume-packages/install-use-packages-dotnet-cli.md

```diff
@@ -63,30 +65,32 @@ dotnet add package Newtonsoft.Json --version 12.0.1
 
 ## List package references
 
-You can list the package references for your project using the [dotnet list package](/dotnet/core/tools/dotnet-list-package?tabs=netcore2x) command.
+List the package references and versions for your project by using the [dotnet list package](/dotnet/core/tools/dotnet-list-package) command:
 
 ```dotnetcli
 dotnet list package
 ```
 
 ## Remove a package
 
-Use the [dotnet remove package](/dotnet/core/tools/dotnet-remove-package?tabs=netcore2x) command to remove a package reference from the project file.
+Use the [dotnet remove package](/dotnet/core/tools/dotnet-remove-package) command to remove a package reference from the project file.
 
 ```dotnetcli
-dotnet remove package <PACKAGE_NAME>
+dotnet remove package <PackageName>
```

Comment:
We should keep the same parameters as the CLI:

https://docs.microsoft.com/en-us/dotnet/core/tools/dotnet-remove-package#arguments